AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/services/allocate-hearings.service.spec.ts

```diff
@@ -63,4 +64,21 @@ describe('AllocateHearingsService', () => {
             expect(result).toEqual(allocationResponse);
         });
     });
+
+    describe('setAllocationToHearings', () => {
+        it('should call the api and return updated hearings with allocated users', fakeAsync(() => {
```

Comment:
The test description says it returns updated hearings with allocated users but we dont mock or assert on any return values in the test.